Templates/project_collection/README.md

```diff
@@ -0,0 +1,212 @@
+  
+
+## Data collection
+This sheet contains information about your data collection
+
+### **Collection** - *collection name*
+  * This is a unique human-readable identifier (i.e. handle) used to keep track 
+    of your data collection
+  * This field is controlled by the vocabulary in the `collection` column on `values` sheet
+  * If this is a new data collection, please email 
+    [data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org)
+    and let us know.
+  * Collection names must be unique, so make sure any new data collections do 
+    not have a name that is already in the controlled vocabulary
+  * By convention, we prefer to use names in `snake_case` without `_proj` as a suffix
+  * This field corresponds to the `has output` field on the `Data collection project` tab
+    
+#### **has samples of type** - *specimen type*
+  * This is the type of specimen that this data collection is about
+  * This field is controlled by the vocabulary in the `specimen type` column on `values` sheet
+  * Each data collection should only have one specimen type
+  * Please choose the lowest-level specimen type that is appropriate for your collection
+  * If you believe that there is more than one specimen type, you might need a new data collection
+  * If you cannot find an appropriate specimen type, or you are not sure which one to use,
+    please email [data.curation@alleninstitute.org](mailto:data.curation@alleninstitute.org)
+    and we will add a new specimen type or help you choose the appropriate one
+    
+#### **is data about** - *species*
+  * This is the common name of the species that this data collection is about
+  * This field is controlled by the vocabulary in the `species` column on `values` sheet
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually there is only one species associated with a data collection
+  * Data collections about human subjects should be separate from other species
+  
+#### **is data of type** - *modality*
+  * This is the name of the data modality for this collection
+  * This field is controlled by the vocabulary in the `modality` column on `values` sheet
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+  * Usually there is only one modality associated with a data collection
+  
+#### **is subject of resource** - *URL*
+  * This is usually a link to the repository where the raw data is stored.
+  * You can submit multiple values for this field, separated by commas or 
+    new rows immediately after this one
+
+#### **was collected using** - *technique*
+  * This is the name technique used to acquire data for this collection
+  * This field is controlled by the vocabulary in the `technique` column on `values` sheet
+  * You can submit multiple values for this field, separated by commas or 
```

Comment:
Need to tell them how to request new techniques

Reply:
Added bullet point telling them to email us to request new technique